json/upickle/src/main/scala/tapir/upickle/TapirJsonuPickle.scala

```diff
@@ -0,0 +1,28 @@
+package tapir.json.upickle
```

Comment:
The file path does not match the package name. Should be `tapir/json/upickle` I guess